src/code.sh

```
@@ -3,10 +3,22 @@
 # The following line causes bash to exit at any point if there is any error
 # and to output each line as it is executed -- useful for debugging
 set -e -x -o pipefail
+# prefixes all lines of commands written to stdout with datetime
+PS4='\000[$(date)]\011'
+export TZ=Europe/London
+
+# set frequency of instance usage in logs to 10 seconds
+kill $(ps aux | grep pcp-dstat | head -n1 | awk '{print $2}')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)